src/pkg/projects/Microsoft.NETCore.App/netcoreapp1.1.props

```diff
@@ -28,6 +28,12 @@
     <NETCoreApp11Dependency Include="NETStandard.Library">
```

Comment:
I would suggest diffing the nuspect for the 1.1.2 package with your local one to make sure the deps are updated.